lib/panko/attribute.rb

```diff
@@ -4,9 +4,17 @@ module Panko
   class Attribute
     def self.create(name, alias_name: nil)
       alias_name = alias_name.to_s unless alias_name.nil?
+      alias_name = transform_key(name.to_s) if alias_name.nil? && Panko.configuration.key_type
       Attribute.new(name.to_s, alias_name)
     end
 
+    def self.transform_key(name)
```

Comment:
can we handle the case where `key_type` is nil and just return the name as is?

Reply:
and looks like I have added check before method calling `if Panko.configuration.key_type`, but I'll add in any case

Reply:
and I think about memoization at this point, what do you think?

if developer serializes 1000 elements with the same key, no need to transform key 1000 times, 1 is enough

Reply:
1. Why we need the `if Panko.configuration.key_type` in `self.create` , we already handle in `transform_key` now.
2. `and I think about memoization at this point, what do you think?` - I don't think it's necessary, since attributes are creating when serializer is defined and not during serialization time, so you pay the "transformation cost" only at boot time.

Reply:
1. Why we need the `if Panko.configuration.key_type` in `self.create` , we already handle in `transform_key` now.
2. `and I think about memoization at this point, what do you think?` - I don't think it's necessary, since attributes are creating when serializer is defined and not during serialization time, so you pay the "transformation cost" only at boot time.